components/heading.tsx

```diff
@@ -77,34 +77,31 @@ export default function Heading({
   href?: string | UrlObject;
   text: string;
 }) {
-  const styles = 'flex flex-nowrap gap-2 sm:gap-3 md:gap-4';
-
   return (
     <header
       className={cn(
+        'group relative',
         'my-4 sm:my-6 md:my-8 lg:my-10 xl:my-12',
-        !href && styles,
+        'flex flex-nowrap items-center space-x-2 sm:space-x-3 md:space-x-4',
```

Comment:
Why were these converted to `space-*` from `gap-*`? Please give explicit reasons for changing things that are not in the scope of the current PR.

Reply:
gap cannot be overridden and introduced an unwanted space at the end of the commit, this is the smallest way to counteract that